k6/demo-test.ts

```diff
@@ -0,0 +1,94 @@
+import { check, sleep } from "k6";
+import http from "k6/http";
+import type { Options } from "k6/options";
+
+/**
```

Comment:
Is this demo-test file required for load test to work?

Reply:
It is not about requirement, there I have example code for different types of testing.

Reply:
Okay, if this code is just for reference use, then I think we should keep it somewhere else either locally or in a GitHub discussion/issue and add a comment in code with reference link of that discussion/issue.

Also check sonarcloud issues if that also we can resolve.

Reply:
While I understand what you say but I believe until and unless everyone are on the same page with these tests we should have it directly in the code so people can refer directly rather than searching here and there (which I know most of the people won't do). I hope this clears my intention of having it next to the other test files.

Reply:
Regarding sonarcloud, it is complaining about the commented code (demo-test.ts), I will suppress it in the evening.

Reply:
Okay shivam, let's keep this file then.

Reply:
Appreciate it.